### ISSUE TITLE
Colour of search button in navbar looks pretty bad

### ISSUE THREAD

**Issue by Roshanjossey** [Roshanjossey/GitHub-old-UI#2]
![image](https://user-images.githubusercontent.com/8488446/41302601-83977f0c-6e88-11e8-8221-3176a726c2b4.png)

It'd be better to change this to a light version of it.

Note: There are preferences to set colour of navbar. This button's view should be handled in those scenarios as well

**Comment by sumayaSalihs** [comment]
@Roshanjossey  I would like to work on this issue.

**Comment by Roshanjossey** [comment]
@sumayaSalihs, excellent. Let me know if you need any help or if you get blocked.

**Comment by sumayaSalihs** [comment]
@Roshanjossey sure I will keep you in the loop.

**Comment by sumayaSalihs** [comment]
@Roshanjossey I am having issues changing the color of: 
<img width="143" alt="screen shot 2018-06-15 at 2 39 22 pm" src="https://user-images.githubusercontent.com/33211695/41464072-f6ecb1ac-70a9-11e8-94d6-cae749a34d59.png">

I realized it was an svg image so I tried using the `fill` css attribute to change the color but it won't change because the image seem to have been embedded in the extension and not the source code download from the project's repository. I will appreciate if you could assist me on this as I am stuck.

**Comment by Roshanjossey** [comment]
@sumayaSalihs, yeah, this image is fetched from GitHub. We don't have any icons in our repository.

One solution I can think of is to create our own svg file with a different colour and serve it form assets.
We can make the existing image transparent to hide it and put the one we create in the background.

I've inverted the colours of the one that's showing right now.

```
<svg width="19" height="20" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
  <defs>
    <rect id="a" width="19" height="20" rx="3"/>
  </defs>
  <g fill="none" fill-rule="evenodd">
    <use fill="#979A9C" xlink:href="#a"/>
    <rect stroke="#5F6165" x=".5" y=".5" width="18" height="19" rx="3"/>
    <path fill="#5F6165" d="M11.76 5.979l-3.8 9.079h-.91l3.78-9.08z"/>
  </g>
</svg>
```

You might want to experiment with the colour a bit more to make is look better.

**Comment by sumayaSalihs** [comment]
@Roshanjossey thats fantastic. I will experiment with the color.

**Comment by sumayaSalihs** [comment]
@Roshanjossey After achieving the transparency of the black search icon 
![screen shot 2018-06-19 at 9 37 08 pm](https://user-images.githubusercontent.com/33211695/41614555-def1af54-7409-11e8-91c3-ca1b18f4d7ad.jpg) 
Implementing the generated svg file in the background generates a 404 error(Not found):


<img width="626" alt="screen shot 2018-06-19 at 9 36 39 pm" src="https://user-images.githubusercontent.com/33211695/41614683-25149fbe-740a-11e8-82b8-f00a65d9cd35.png">



<img width="636" alt="screen shot 2018-06-19 at 9 37 30 pm" src="https://user-images.githubusercontent.com/33211695/41614692-2e885d60-740a-11e8-8f1f-349380e2a350.png">


css snapshot: 
<img width="538" alt="screen shot 2018-06-19 at 9 36 17 pm" src="https://user-images.githubusercontent.com/33211695/41614755-55a8ef7c-740a-11e8-9be3-c541a6d9c055.png">

I will appreciate if you could assist me on this as I am stuck.

**Comment by Roshanjossey** [comment]
I'll help you to understand the problem first. In the next comment, I'll share links to solution too. But before going there, I'd like to you to search for solutions yourself.

Look at the URL that's getting 404. It's trying to fetch from GitHub. It's obviously not there. It's in the extensions files. So the background-image URL is wrong. You gotta fix that.

Also note: Background image can't be an HTML file (I see extension .html). We decided to go with SVG, didn't we?

**Comment by Roshanjossey** [comment]
Here are some links to find solutions for the problem

https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/web_accessible_resources

https://stackoverflow.com/questions/3559781/google-chrome-extensions-cant-load-local-images-with-css

**Comment by sumayaSalihs** [comment]
@Roshanjossey yes it was supposed to be an svg file but I name both the svg file and html file the same while experimenting with the colors so autocomplete just completed with the html and I didn't even notice.

<img width="375" alt="screen shot 2018-06-20 at 9 26 42 am" src="https://user-images.githubusercontent.com/33211695/41638770-27af8614-746c-11e8-9a55-d53f565df657.png">

I definitely see what is going wrong now.

**Comment by Roshanjossey** [comment]
@sumayaSalihs, how's it going? Would you like to submit a pull request while you're working on the issue? This will help us fast-track the review process. I'll be able to try out your changes easily too.

**Comment by sumayaSalihs** [comment]
@Roshanjossey it is going well but I keep getting error even after making the svg image web accessible in the manifest.js and setting it to the background image in the content.js. I will submit a pull request right away so you can track the changes I make.

**Comment by Roshanjossey** [comment]
Closing this as this has been resolved in #3
